pkg/registry/ecr.go

```diff
@@ -107,6 +107,7 @@ func (e *ECRClient) buildEcrTags() []*ecr.Tag {
 	ecrTags := []*ecr.Tag{}
 
 	for _, t := range e.tags {
+		t := t
```

Comment:
Why is this required?

Reply:
If we have multiple tags, then as it's a pointer we will add the same tag multiple times. I added tests for this. You can try without this line, you'll see it failing :-)

Reply:
this should not be required anymore when `aws.String()` is used.